osxkeychain/osxkeychain_darwin.go

```diff
@@ -134,8 +134,21 @@ func (h Osxkeychain) List() (map[string]string, error) {
 	return resp, nil
 }
 
+// alwaysHTTPS takes a url and makes sure it always has the https protocol
+func alwaysHTTPS(url string) string {
```

Comment:
I modified this to strip any protocol (regex: `^.*//`) with https.  Does that work?